app/manager-frontend/src/components/forms/FormikSelect.jsx

```diff
@@ -5,40 +5,33 @@ import {
   Select,
   Tooltip,
   Icon,
+  Alert,
+  AlertIcon,
+  VStack,
 } from "@chakra-ui/react";
 import { useField } from "formik";
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import { FormHelpError } from "./FormHelpError";
-import { AiFillInfoCircle } from "react-icons/ai";
+import { FaInfoCircle } from "react-icons/fa";
 
 export const FormikSelect = ({
   name,
   label,
   options,
-  placeholder,
-  type = "select",
   isRequired,
   fieldTip,
   fieldHelp,
   collapseError,
-  sourceList,
-  sourceParam,
   tooltip,
-  warning,
-  ...p
+  alert,
+  hasEmptyDefault,
 }) => {
-  const [field, meta, helpers] = useField({ name, type });
+  const [field, meta, helpers] = useField({ name, type: "select" });
   const [value, setValue] = useState(field.value);
 
   const handleChange = ({ target: { value } }) => {
-    let formikValue = value;
-    if (sourceList && sourceParam) {
-      formikValue = sourceList.find((item) => item[sourceParam] == value);
-      setValue(value);
-    } else {
-      setValue(value);
-    }
-    helpers.setValue(formikValue);
```

Comment:
Removing this will break how the select input works when editing a results modifier, as it needs to be able to set the formik value to a whole object rather than just a string value. The code shouldn't change how the component works unless the source list and source param values are set and it still sets the of the input element to the string value when they are set.

Reply:
I can fix all that in a later PR tho? As it doesn't matter for anything that you can currently access in the frontend

Reply:
I thought it was half and half on string vs object values, so we just simplified everything to string for the sake of formik and submission:

- validation on formik value is just a string: https://github.com/link-lite/hutch/blob/d79c673c0706622c97faf4bc43cdfea0cab67af0/app/manager-frontend/src/pages/ActivitySource/validation.js#L12
- initial formik value is jsut a string, and is looked up from the existing activity source's string property when editing: https://github.com/link-lite/hutch/blob/d79c673c0706622c97faf4bc43cdfea0cab67af0/app/manager-frontend/src/pages/ActivitySource/index.jsx#L111
- backend expects a string: https://github.com/link-lite/hutch/blob/d79c673c0706622c97faf4bc43cdfea0cab67af0/app/HutchManager/Models/CreateActivitySource.cs#L11

Reply:
by all means please test it locally to confirm